src/View.Uno/Converters/PrettyDistanceConverter/FromDistanceToPrettyStringConverter.cs

```diff
@@ -52,7 +56,7 @@ public object Convert(object value, Type targetType, object parameter, GenericCu
 				throw new ArgumentException("This converter does not use any parameters");
 			}
 
-#if !WINDOWS_UWP && !__ANDROID__ && !__IOS__ && !__WASM__
+#if !WINDOWS_UWP && !__ANDROID__ && !__IOS__ && !__MACOS__ && !__WASM__ && !__WINDOWS10_0_18362_0__
```

Comment:
please make sure to have a blocked task or wtv to remind us we need to move these under winui too